server/web.go

```diff
@@ -84,6 +84,7 @@ func (provider *WebProvider) Provide(configurationChan chan<- types.ConfigMessag
 
 	// ping route
 	systemRouter.Methods("GET").Path(provider.Path + "ping").HandlerFunc(provider.getPingHandler)
+	systemRouter.Methods("HEAD").Path(provider.Path + "ping").HandlerFunc(provider.getPingHandler)
```

Comment:
Could you instead use `systemRouter.Methods("GET", "HEAD").Path(provider.Path + "ping").HandlerFunc(provider.getPingHandler)` ?